notebooks/4_7_using_the_memtile_in_your_applications.ipynb

```diff
@@ -729,7 +729,7 @@
     "            x_out[row] = mtbout\n",
     "```\n",
     "\n",
-    "In this example, `MTPassThrough()` only buffers data in the local memory of the memory tile. It does not carry out any operation on the data. `self.mtbuffer_in()` buffers the input tile in the memory tile before sending it to the compute tile, the `self.mtbuffer_out()` buffers the output data in the memory tile before it is streamed out of the arraying. Note that, `mtpin` and `mtpout` are references to the buffers that are allocated in the memory tile.\n",
+    "In this example, `MTPassThrough()` only buffers data in the data memory of the memory tile. It does not carry out any operation on the data. `self.mtbuffer_in()` buffers the input tile in the memory tile before sending it to the compute tile, the `self.mtbuffer_out()` buffers the output data in the memory tile before it is streamed out of the array. Note that, `mtpin` and `mtpout` are references to the buffers that are allocated in the memory tile.\n",
```

Comment:
I'm not sure if saying data memory of the memtile here might be a little bit confusing, it might imply that there is instruction memory. How about just leaving it as memory?